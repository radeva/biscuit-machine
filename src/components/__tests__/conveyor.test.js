import React from 'react';
import Conveyor from '../conveyor';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

describe('Conveyor', () => {
  it('should render correctly when off', () => {
    const { container } = render(<Conveyor isOn={false} />);

    expect(container).toMatchSnapshot();
  });

  it('should render correctly when on', () => {
    const { container } = render(<Conveyor isOn={true} />);

    expect(container).toMatchSnapshot();
  });
});
