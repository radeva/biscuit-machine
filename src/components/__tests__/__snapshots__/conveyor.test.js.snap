// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders conveyor correctly when off 1`] = `
<div>
  <div
    class="conveyor"
  >
    <svg>
      conveyor.svg
    </svg>
  </div>
</div>
`;

exports[`renders conveyor correctly when on 1`] = `
<div>
  <div
    class="conveyor"
  >
    <svg
      class="sc-AxjAm eBcJJU"
    >
      conveyor.svg
    </svg>
  </div>
</div>
`;
